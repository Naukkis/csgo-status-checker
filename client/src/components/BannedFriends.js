import React from 'react';
import PropTypes from 'prop-types';

const BannedFriends = props => (
  props.bannedFriends > 0 ?
    <p style={{ color: 'red' }}>Banned friends: {props.bannedFriends}</p>
    :
    <p>Banned friends: {props.bannedFriends}</p>
);

BannedFriends.propTypes = {
  bannedFriends: PropTypes.number.isRequired,
};

export default BannedFriends;
