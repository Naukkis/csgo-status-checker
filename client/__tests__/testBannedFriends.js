import React from 'react';
import renderer from 'react-test-renderer';
import BannedFriends from '../src/components/BannedFriends';

it('renders correctly', () => {
  const tree = renderer
    .create(<BannedFriends bannedFriends={12} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
