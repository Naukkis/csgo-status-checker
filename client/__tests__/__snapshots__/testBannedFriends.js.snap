// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  style={
    Object {
      "color": "red",
    }
  }
>
  Banned friends: 
  12
</p>
`;
